### PR TITLE
Added a handler for rejected promises 

### PR DESCRIPTION
**Pull request by katelynsills**
## Description
This is in response to this message: 

```DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

Added a handler function for the rejected promises (any async route functions with errors are rejected promises). Fixed some settings to be able to run the tests in order to ensure that the handler works.

## Test Plan
yarn test

I also threw a new Error in one of the routes and ensured that the error was logged, it returned a 500 status code, and that we didn't get the alarming message anymore.

### CODE REVIEW

File: config/test.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ module.exports = {
 		name: 'hah-test'
 	},
 	facebook: {
-		appId: process.env.FACEBOOK_APP_ID,
-		appSecret: process.env.FACEBOOK_APP_SECRET,
+		appId: process.env.V_FACEBOOK_APP_ID,
+		appSecret: process.env.V_FACEBOOK_APP_SECRET,
```
Comment (karenmrose) [inline config/test.js]: Just out of curiosity, what is the V for?

Comment (katelynsills) [inline config/test.js]: That was to distinguish between the Verification portal Facebook app, and any other Facebook app we might need, but it's looking now like we won't need another app, so it's probably unnecessary.

Comment (karenmrose) [inline config/test.js]: Ah ok. We may still need one in the future for people who want to sign in and add their local resources. What do you think about making it more descriptive though `VERIFICATION_PORTAL_FACEBOOK_APP_ID` or something like that? I know that's long, but I'm definitely a fan of verbose/descriptive names over short names. Thoughts?

File: server/server.js [hunk 2]
```diff
@@ -73,6 +73,11 @@ passport.deserializeUser((objectId, done) => {
 	})
 })
 
+// adapted from https://strongloop.com/strongblog/async-error-handling-expressjs-es7-promises-generators/
```
Comment (karenmrose) [inline server/server.js]: I actually really like the comment in this case, that was helpful 😉

File: config/test.js (hunk 1 above)
Comment (katelynsills) [inline config/test.js]: Sounds good, I went ahead and made that change

**Comment by karenmrose** [comment]
This looks really good, nice work! The only thing I would ask is what do you think about changing `wrap` to something that is a little more descriptive of what it's doing/handling?

**Comment by katelynsills** [comment]
Sure, what kind of thing are you thinking? Something like 'handleRejectedPromise'?

**Comment by karenmrose** [comment]
Yeah that works!

**Pull request merged**